receiver/datadogreceiver/receiver.go

```diff
@@ -32,27 +32,43 @@ func newDataDogReceiver(config *Config, nextConsumer consumer.Traces, params rec
 	if err != nil {
 		return nil, err
 	}
+
 	return &datadogReceiver{
 		params:       params,
 		config:       config,
 		nextConsumer: nextConsumer,
 		server: &http.Server{
 			ReadTimeout: config.ReadTimeout,
-			Addr:        config.HTTPServerSettings.Endpoint,
```

Comment:
Just to confirm, this is no longer needed because we are using the listener to bind the interface?